roles/etcd_client_certificates/README.md

```diff
@@ -1,5 +1,5 @@
-OpenShift etcd certificates
-========================
+OpenShift etcd Client Certificates
+==================================
```

Comment:
What do you think about consolidating etcd_ca, etcd_client_certificates, and etcd_server_certificates?

tasks/main.yaml could do conditional includes based on a variable describing which action to take.